tests/integration/test_logcollector/test_log_filter_options/test_ignore_regex.py

```diff
@@ -0,0 +1,357 @@
+TEST_CASES_PATH = os.path.join(TEST_DATA_PATH, 'test_cases')
+
+# Test configurations and cases data
+test_file = os.path.join(PREFIX, 'test')
+# --------------------------------TEST_IGNORE_DEFAULT--------------------------------------------
+t1_configurations_path = os.path.join(CONFIGURATIONS_PATH, 'configuration_ignore_regex_default.yaml')
+t1_cases_path = os.path.join(TEST_CASES_PATH, 'cases_ignore_regex_default.yaml')
+
+t1_configuration_parameters, t1_configuration_metadata, t1_case_ids = get_test_cases_data(t1_cases_path)
+for count, value in enumerate(t1_configuration_parameters):
+    t1_configuration_parameters[count]['LOCATION'] = test_file
+t1_configurations = load_configuration_template(t1_configurations_path, t1_configuration_parameters,
+                                                t1_configuration_metadata)
+
+# --------------------------------TEST_IGNORE_REGEX_TYPE_VALUES--------------------------------------
+t2_configurations_path = os.path.join(CONFIGURATIONS_PATH, 'configuration_ignore_regex_type_values.yaml')
+t2_cases_path = os.path.join(TEST_CASES_PATH, 'cases_ignore_regex_type_values.yaml')
+
+t2_configuration_parameters, t2_configuration_metadata, t2_case_ids = get_test_cases_data(t2_cases_path)
+for count, value in enumerate(t2_configuration_parameters):
+    t2_configuration_parameters[count]['LOCATION'] = test_file
+t2_configurations = load_configuration_template(t2_configurations_path, t2_configuration_parameters,
+                                                t2_configuration_metadata)
+
+# --------------------------------TEST_IGNORE_MULTIPLE_REGEX-------------------------------------------
+t3_configurations_path = os.path.join(CONFIGURATIONS_PATH, 'configuration_ignore_multiple_regex.yaml')
+t3_cases_path = os.path.join(TEST_CASES_PATH, 'cases_ignore_multiple_regex.yaml')
+
+t3_configuration_parameters, t3_configuration_metadata, t3_case_ids = get_test_cases_data(t3_cases_path)
+for count, value in enumerate(t3_configuration_parameters):
+    t3_configuration_parameters[count]['LOCATION'] = test_file
+t3_configurations = load_configuration_template(t3_configurations_path, t3_configuration_parameters,
+                                                t3_configuration_metadata)
+
+prefix = lc.LOG_COLLECTOR_PREFIX
+local_internal_options = lc.LOGCOLLECTOR_DEFAULT_LOCAL_INTERNAL_OPTIONS
+
+
+# Tests
+@pytest.mark.tier(level=0)
+@pytest.mark.parametrize('new_file_path,', [test_file], ids=[''])
+@pytest.mark.parametrize('configuration, metadata', zip(t1_configurations, t1_configuration_metadata), ids=t1_case_ids)
+def test_ignore_default(configuration, metadata, new_file_path, create_file, truncate_monitored_files,
+                        set_wazuh_configuration, configure_local_internal_options_function, restart_wazuh_function):
+    '''
+    description: Check if logcollector reads or ignores a log according to a regex configured in the ignored tag for a
+                 given log file.
+
+    test_phases:
```

Comment:
Use the new documentation model about this tag

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/pull/3582/commits/51386cf575df408432ac4c69f308c77477006e9e